plugins/lua/lua_plugin.c

```diff
@@ -983,7 +983,7 @@ static int uwsgi_api_register_signal(lua_State *L) {
 		uwsgi_lock(uwsgi.signal_table_lock);
 
 		// master-proc checks receiver
-		strcpy(use->receiver, who);
+		strncpy(use->receiver, who, len);
```

Comment:
Sorry, for slow reaction, but
```c
strncpy(use->receiver, who, len);
```
lua provides len without terminator, so we need do
```c
strncpy(use->receiver, who, len + 1); //also copy terminator from 'who'
```
or
```
strcpy(use->receiver, who); //len check has been passed, 'who' has terminator
```